testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/client/CredentialsTest.java

```diff
@@ -131,7 +131,7 @@ public void testUploadKeyAndCertificate() throws Exception {
             FileInputStream fs = new FileInputStream(generatedKeystore.getKeystoreFile());
             byte [] content = fs.readAllBytes();
             fs.close();
-            keyCertForm.addFormData("file", content, MediaType.APPLICATION_OCTET_STREAM_TYPE);
+            keyCertForm.addFormData("file", content, MediaType.APPLICATION_OCTET_STREAM_TYPE, "keys");
```

Comment:
I wonder, now that we need to specify a file name (`keys` in this case), isn't it a breaking change?

Reply:
Also specific to Resteasy Reactive, I can remove it from here and send in the next PR.